playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss

```diff
@@ -0,0 +1,111 @@
+@import "../tokens/positioning";
+@import "../tokens/colors";
+@import "../pb_card/card_mixin";
+@import "../tokens/shadows";
+
+@keyframes modalFadeIn {
+	from {
+		transform: translate3d(0, -100%, 0);
+		opacity: 0;
```

Comment:
Indentation is huge. What is your setting?

Reply:
Oops. For some reason, tabs was set to 4. Fixed.